test/system/flows/admin/admin_views_mentor_discussion_test.rb

```diff
@@ -0,0 +1,34 @@
+require "application_system_test_case"
+require_relative "../../../support/capybara_helpers"
+
+module Flows
+  module Admin
+    class AdminViewsMentorDiscussionTest < ApplicationSystemTestCase
+      include CapybaraHelpers
+
+      test "admin views mentor discussion but is not student nor mentor" do
```

Comment:
I've added a system test to verify that admins can view mentor discussions. I should probably also add this for viewing exercises at some point.

For now, I've put this test in a separate `admin` directory (we already have `mentor` and `student` directories). Let me know if this is fine or if you'd like to see it moved.